src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -102,6 +102,9 @@ def cluster_projects(projects: Sequence[Project]) -> None:
                 tags={"clustered": False},
                 sample_rate=1.0,
             )
+            sentry_sdk.set_tag("projs.total", len(projects))
+            sentry_sdk.set_tag("projs.unclustered", unclustered)
```

Comment:
I'm not sure if we want to use tags for this,  I would rather use `set_extra` and would give it longer names, meaning `projs` -> `projects`

Reply:
Tags are queryable, so they allow us to evaluate how many projects were unclustered. Using the context would require opening all events individually and summing them up one by one. The number of unclustered projects is useful to evaluate the impact.

Reply:
Regarding the tag names, I'm open to updating them but can make queries too long and is easy to understand as-is. What do you think?

Reply:
I still prefer proper names, event if they are longer.